### ISSUE TITLE
Significant alignment issues on Resources page on mobile

### ISSUE THREAD

**Issue by mark-a-wilson** [bcgov/digital.gov.bc.ca#289]
**Note:** cross-reference #359, #362

See [this screen recording](https://drive.google.com/file/d/1K82wVIZfYTgm5hN5iQnQaLR5SivDFZXX/view?usp=sharing) of viewing the **Resources** page ([this deployment](https://digital-gov-frontend-pr-199-xdwiwd-dev.pathfinder.gov.bc.ca/resources)) in Safari on iPhone 7.

### Acceptance Criteria

All sections of this page are aligned and do not overflow the display when viewed on mobile.

**Comment by thegentlemanphysicist** [comment]
There's a good chance this gets resolved once #364 is merged into develop

**Comment by mark-a-wilson** [comment]
Yes, looks like it 👍

**Comment by mark-a-wilson** [comment]
Closing as this has been resolved by #364 which is now merged into develop.
